jobs/future-effective-filings/devops/vaults.gcp.env

```diff
@@ -1,7 +1,7 @@
 BUSINESS_API_URL="op://API/$APP_ENV/business-api/BUSINESS_API_URL"
 BUSINESS_API_VERSION2="op://API/$APP_ENV/business-api/BUSINESS_API_VERSION_2"
-AUDIENCE=op://gcp-queue/$APP_ENV/payment/AUDIENCE"
-TOPIC_NAME=op://gcp-queue/$APP_ENV/payment/TOPIC_NAME"
-PUBLISHER_AUDIENCE=op://gcp-queue/$APP_ENV/payment/PUBLISHER_AUDIENCE"
-GCP_AUTH_KEY=op://gcp-queue/$APP_ENV/payment/GCP_AUTH_KEY"
+AUDIENCE="op://gcp-queue/$APP_ENV/payment/AUDIENCE"
+TOPIC_NAME="op://gcp-queue/$APP_ENV/payment/TOPIC_NAME"
+PUBLISHER_AUDIENCE="op://gcp-queue/$APP_ENV/payment/PUBLISHER_AUDIENCE"
+GCP_AUTH_KEY="op://gcp-queue/$APP_ENV/payment/GCP_AUTH_KEY"
```

Comment:
everything is linked to the `payment` vault.  is this right?

Reply:
Was just wondering about that. My first guess would be yes. The Audience is always "https://pubsub.googleapis.com/google.pubsub.v1.Subscriber" and the PUBLISHER_AUDIENCE is always "https://pubsub.googleapis.com/google.pubsub.v1.Publisher". The same is for GCP AUTH KEY. All is good.

Now, the TOPIC_NAME, I think should be different.

Reply:
there is this, so we could use that 
![image](https://github.com/bcgov/lear/assets/6685223/279fe31f-57ba-4467-9dca-97ea371c5418)

Reply:
Yup, exactly. Thanks!